### ISSUE TITLE
[1.16.4] Recipes that use the Chambers should show when using U on the Chamber

### ISSUE THREAD

**Issue by aaronhowser1** [PrinceOfAmber/Cyclic#1664]
Minecraft Version: 1.16.4

Forge Version: 35.1.7

Mod Version: Cyclic-1.16.4-1.0.2

Single Player or Server: SP

Describe problem (what you were doing; what happened; what should have happened):
Most recipes show the blocks that they use on the left in JEI. They can also be seen by using U on that block. Using U on a furnace, for instance, shows this:
![image](https://user-images.githubusercontent.com/15698722/103478991-eefaad80-4d8f-11eb-863e-3b0fa9809391.png)

However, Cyclic machines do not.
![image](https://user-images.githubusercontent.com/15698722/103478963-cbcffe00-4d8f-11eb-8c23-d2d19c4e6974.png)
This is what you get when you hit U on the Melting Chamber, for instance:
![image](https://user-images.githubusercontent.com/15698722/103479025-24070000-4d90-11eb-815f-aac64a9d91d3.png)

Basically, the Chambers should show up as a "catalyst", or whatever the name is, for the recipe. Essentially every other mod does this.
